.circleci/config.yml

```diff
@@ -1105,11 +1105,11 @@ jobs:
   # Builds in C++20 mode and uses debug build in order to speed up.
   # Do *NOT* store any artifacts or workspace as we don't run tests on this build.
   b_ubu_cxx20:
-    <<: *base_ubuntu2404_large
+    <<: *base_ubuntu2404_xlarge
     environment:
-      <<: *base_ubuntu2404_large_env
+      <<: *base_ubuntu2404_xlarge_env
       CMAKE_BUILD_TYPE: Debug
-      CMAKE_OPTIONS: -DCMAKE_CXX_STANDARD=20
+      CMAKE_OPTIONS: -DCMAKE_CXX_STANDARD=20 -DCMAKE_CXX_FLAGS=-fuse-ld=gold
```

Comment:
linker flags are not cxx flags... I think `CMAKE_EXE_LINKER_FLAGS` should be used here

Reply:
Are you sure? This is what ends up in the CI
```
cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_STANDARD=20 -DCMAKE_CXX_FLAGS=-fuse-ld=gold -G 'Unix Makefiles'
```
And I see no complaints regarding the flag.

Reply:
I think it will not really use it for linking.. maybe you could check with `make VERBOSE=1`?

Reply:
Yup, I'd say you're right.

Reply:
I just saw https://github.com/ethereum/solidity/pull/15474 - so maybe its really fine with gcc.. but its strange